less/index.less

```diff
@@ -57,3 +57,172 @@ html, body {
     height: 100%;
 }
 
+
+//Variables
+@color:#D3D3D3;
+@size:30px;
+
+//Mixin
+.position(){
+	font-size:size;
```

Comment:
It'd be best to use `rem` units here for `font-size` since it scales to users browser font settings.